Anytype/Sources/PresentationLayer/ObjectCreationSettings/Views/Selection/Dataview/SetObjectCreationSettingsInteractor.swift

```diff
@@ -147,7 +147,9 @@ final class SetObjectCreationSettingsInteractor: SetObjectCreationSettingsIntera
     private func updateObjectTypes() {
         let objectTypes = objectTypesProvider.objectTypes(spaceId: setDocument.spaceId).filter {
             guard let recommendedLayout = $0.recommendedLayout else { return false }
-            return !$0.isArchived && DetailsLayout.visibleLayouts.contains(recommendedLayout)
+            return !$0.isArchived &&
+            DetailsLayout.visibleLayouts.contains(recommendedLayout) &&
+            $0.uniqueKey != ObjectTypeUniqueKey.template
```

Comment:
maybe use isTemplateType field in object details?

Reply:
let me check

Reply:
move isTemplateType to the `ObjectType`